src/foam/core/FObject.js

```diff
@@ -129,6 +129,16 @@ foam.LIB({
        */
       this.private_.axiomCache = {};
 
+      // Sort axioms by priority, higher priority gets installed first.
+      // Default to 100.
+      axs = axs.sort(function(a, b) {
+        var p1 = foam.Number.isInstance(a.priority) ? a.priority : 100;
+        var p2 = foam.Number.isInstance(b.priority) ? b.priority : 100;
+
+        // compare p2 vs p1, as we want higher priority values first.
+        return foam.Number.compare(p2, p1);
```

Comment:
Would this work?
return foam.Number.compare(b.priority || 100, a.priority || 100);

Reply:
Yes, it works. Updated.